docs/antctl.md

```diff
@@ -679,3 +681,33 @@ worker1 172.18.0.4 Alive
 worker2 172.18.0.3 Alive 
 worker3 172.18.0.2 Dead
 ```
+
+### Upgrade existing objects of CRDs
+
+antctl supports upgrading existing objects of Antrea CRDs to the latest version.
```

Comment:
```suggestion
antctl supports upgrading existing objects of Antrea CRDs to the storage version.
```

Reply:
Should we change all `the latest version` to `the storage version`? Generally, the `storage version` should be the latest version. @tnqn

Reply:
Done